### DISCUSSION TITLE
Meaning of load_power_r/s/t

### DISCUSSION THREAD

**Discussion by xrad** [nathanmarlor/foxess_modbus#688] [Q&A]
What the title says. Clearly it is not the house load. It seems to be related to the inverter itself, but I would like to know a more exact definition. I'm on H3 with FW 2.01. TIA for any insights!

**Comment by xrad** [comment]
"Clearly" may have a been a bold choice of words given my overal ignorance. But when I look at the graphed sum of these values they keep being in a corridor between about 70 to 190 W when the inverter is on, and 0 W when the inverter is in stand-by. So it _looks_ like it might just the inverter's own consumption. But then again, why is it called "load_power"? Therefore, any insights appreciated!

**Reply by canton7** [reply]
Load is house load, yep. See https://github.com/TonyM1958/HA-FoxESS-Modbus/wiki/How-does-it-work%3F

**Reply by xrad** [reply]
Reading other people's graphs is difficult, so I deleted mine. Instead, here are some current readings using two different kinds of manual load calculation and getting the delta of the two. If I compare that with the inv_load rst sum, it is strikingly close. Sometimes there is a 10-20 W difference, but the point is that inv_load is always very close to what I would assume the inverter consumes. The Load RST values in the Fox cloud however are actually indication the actual house load.

```
PV + Charge - Discharge - CT => 0.377
Inv_Power - CT               => 0.243
Delta                        => 0.134
Inv_Load                     => 0.135
```

**Reply by udochrist** [reply]
I concur with xrad and would also like to chime in because the values is see in the entities do not make sense to use "Load Power" as the house consumption.

Here is a screenshot of the Power entities (N.B.: it would be great if power entities would be named with a power and energy entites would be named "energy" or consumption/production. Currently there are power entities that provide "energy" (kWh) readings)

<img width="401" height="919" alt="Screenshot 2025-07-17 at 14 46 21" src="https://github.com/user-attachments/assets/193a9a81-01a0-4d04-82d8-482fc352ad2f" />

EDIT for clarification : The "inverter Power Usage" is a computed entity and not provided by the integration.


Looking at the entities:
- Battery Charge/Discharge: always positive showing power charging/discharging the battery.
  - Inverter Battery Power seems to be the same but with a direction attached.
- Grid CT RST: the power measurements of the 3phase connected smartmeter to the inverter.
  - Negative seems to indicate we are using power from  the net
  - Positive seems to indicate we are feeding power to the net

- Feed-In RST: only the absolute of the positive values of Grid CT
- Grid Consumption RST: only the absolute of the negative values of Grid CT
- Inverter Power: This must be the "house power consumption" (possibly including the inverter itself). at the time of snapshot the heatpump compressor was running so 4kW is feasible
- Load Power: This cannot be the house consumption. the values are far to low. They look like the inverters "self-usage"
- PVx Power: Total and single string pv power.

My inverter is a Kopp Kuara 12.0-3 H (a foxess derivative).
When Load Power really should be the house consumption, Is it then possible that the modbus addresses are somehow mixed up for inverter and load power? otherwise i don't understand the values read back.
